README.md

```diff
@@ -6,7 +6,7 @@ This project contains Kubernetes manifests for self-deployed OTLP ingest on Kube
 
 Before we begin, set required environment variables:
 ```console
-export GCLOUD_PROJECT=<your project id>
+export PROJECT_ID=<your project id>
```

Comment:
Looking over other documents, it looks like this variable is most commonly referred to as:

 -  `GOOGLE_CLOUD_PROJECT`

... though there are also places in which it is referred to as `GCLOUD_PROJECT`, `GCP_PROJECT`, `DEVSHELL_PROJECT_ID`.

Would it make sense to use one of these other values to increase the odds of it being something already defined for the user?

Also, would it make sense to provide a solution to automatically initialize it from the `gcloud config configurations ...` setting (e.g. via `gcloud config get project 2>/dev/null`)

Reply:
I can switch the variable name back. It was updated to match some internal docs / user guides.

Regarding automatic initialization: I dont have a strong opinion on this, but I think manually setting it makes the process a little transparent for the user.

Reply:
I think `GOOGLE_CLOUD_PROJECT` is the canonical one. I.e. all the google auth libraries specifically read that environment variable to populate the project.